setup.cfg

```diff
@@ -22,7 +22,7 @@ packages = find:
 python_requires = >=3.8
 install_requires =
     click
-    depthcharge-ms @ git+https://github.com/wfondrie/depthcharge@3dda29053ab479fa31fa1309f65f285d04481c1e
+    depthcharge-ms==0.0.1
```

Comment:
**suggestion:** I'd use `>=` to get the latest optional depthcharge improvements without having to update the Casanovo dependencies every time.

Reply:
Whoops! Good catch 😅